exampleSite/config.toml

```diff
@@ -80,6 +80,7 @@ paginate = 10
     default_sharing_image = "img/sharing-default.png"
 
     # Google Maps API key (if not set will default to not passing a key.)
+    enableGoogleMaps = true
```

Comment:
I would add this option to the readme.md

Reply:
Yep, `enableGoogleMaps` should be documented in the readme. Plus the comment on line 82 should be changed accordingly, right now it only refers to `googleMapsApiKey`.

Reply:
Or, since (as far as I know) the Google map only reliably works when a valid `googleMapsApiKey` is provided: Why not change `googleMapsApiKey` to work in the same way as [`googleAnalytics` currently does](https://github.com/devcows/hugo-universal-theme/#google-analytics)? I.e. not setting a Google Maps API key would then have the same result as setting `enableGoogleMaps = false` in this PR.

In `layouts/partials/scripts.html` we could then throw out the lines

```html
{{ else }}
<script src="//maps.googleapis.com/maps/api/js?v=3.exp"></script>
```

and move the `{{ end }} ` statement of the `{{ if .Site.Params.googleMapsApiKey }}` clause two lines down.

Reply:
so, what could we do? 
googleMapsApiKey or enableGoogleMaps & googleMapsApiKey

Reply:
@ryanfox1985 What do you think about my [suggestion](https://github.com/devcows/hugo-universal-theme/pull/260#discussion_r371475318)? Does anything speak against removing the option to use the Google map without a `googleMapsApiKey`? As I said, I _think_ the map is of no use without a valid key, but I might be wrong...

Reply:
there is a use case without token (demo site). I would not remove.

Reply:
> there is a use case without token (demo site). I would not remove.

Ok then, both `enableGoogleMaps` and `googleMapsApiKey` it will be.

So @helight , forget everything in this comment thread except [my first comment](https://github.com/devcows/hugo-universal-theme/pull/260#discussion_r371464103). :)

Reply:
got and done. thanks @salim-b .

Reply:
Thanks. I've just pushed some more style/language improvements.

From my side this is ready to be merged now :)